test/Network/Kontiki/RaftSpec.hs

```diff
@@ -0,0 +1,12 @@
+module Network.Kontiki.RaftSpec where
```

Comment:
I'm very confused by this module in general. What is it testing? Why is Test.QuickCheck, MemLog,... imported?